browser/ui/BUILD.gn

```diff
@@ -1374,6 +1375,23 @@ source_set("brave_tab_features") {
   deps = [ "//base" ]
 }
 
+source_set("brave_rewards_source") {
+  sources = [
+    "webui/brave_rewards_source.cc",
+    "webui/brave_rewards_source.h",
```

Comment:
there's `ui/webui/brave_rewards` dir. Would be nice to move this into it.

Reply:
trying to avoid any changes that aren't necessary here because it already touches a lot of stuff